driver/razerchromacommon.c

```diff
@@ -201,6 +201,50 @@ struct razer_report razer_chroma_standard_get_led_brightness(unsigned char varia
     return report;
 }
 
+/*
+ * Set (three) LED states of Tartarus Pro deivce.
+ * The @param led_state should be fetched by `_get_led_state`, and can only contains either on (0xFF) or off (0x00) for each LED.
+ * NOSTORE.
+ *
+ * Status Trans Packet Proto DataSize Class CMD Args
+ * 00     1f    0000   00    09       0f    02  010b0100010100ff00 | (? 0x010b01000101, RGB 0x00ff00)
+ */
+struct razer_report razer_chroma_tartarus_set_led_state(struct razer_rgb *rgb)
+{
+    struct razer_report report = get_razer_report(0x0F, 0x02, 9);
+    report.arguments[0] = 0x01;
+    report.arguments[1] = 0x0B;
+    report.arguments[2] = 0x01;
```

Comment:
up to here it's equivalent to `razer_chroma_extended_matrix_effect_base(0x09, VARSTORE, 0x0B, 0x01)` 

but `razer_chroma_extended_matrix_effect_static()` does that already for you and seems to be exactly what you need, just with led_id 0x0B apparently, which apparently is called "Side Stripe LED", so make a new define in driver/razercommon.h around `RED_PROFILE_LED` for `#define SIDE_STRIPE_LED 0x0B`

Reply:
Havn't aware of this, thanks for pointing it out! I'll remove those chroma functions.